openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -137,8 +136,8 @@ compiler/intrinsics/mathexact/LongMulOverflowTest.java https://bugs.openjdk.org/
 runtime/containers/docker/DockerBasicTest.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
 runtime/containers/docker/TestCPUAwareness.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
 runtime/containers/docker/TestCPUSets.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
-runtime/containers/docker/TestMemoryAwareness.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
 runtime/containers/docker/TestMisc.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
+runtime/containers/docker/TestMemoryAwareness.java https://bugs.openjdk.java.net/browse/JDK-8229182,https://github.com/adoptium/aqa-tests/issues/3629,https://github.com/adoptium/aqa-tests/issues/3905 generic-all
```

Comment:
I am not sure the comma can be handled by the disabled test pipeline. Maybe we should add support for it.

Reply:
The comma handled by jtreg. The openjdk upstream also use comma:

![image](https://user-images.githubusercontent.com/24123821/182908358-0e284d21-9bfc-49da-9fb3-014f48b5f61a.png)